tests/integration/hooks.bats

```diff
@@ -42,3 +42,41 @@ function teardown() {
 		[[ "$output" == *"error running $hook hook #1:"* ]]
 	done
 }
+
+@test "runc run [hook with env]" {
+	update_config '.process.args = ["/bin/true"]'
+	update_config '.process.env = ["mm=nn"]'
+	# All hooks except Poststop.
+	for hook in prestart createRuntime createContainer startContainer poststart; do
+		echo "testing hook $hook"
+		# shellcheck disable=SC2016
+		update_config '.hooks = {
+			"'$hook'": [{
+				"path": "/bin/sh",
+				"args": ["/bin/sh", "-c", "[ \"$mm\"==\"tt\" ] && echo yes, we got tt from the env mm && exit 1 || exit 0"],
+				"env": ["mm=tt"]
+			}]
+		}'
+		mm=nn runc run "test_hook-$hook"
```

Comment:
Not sure what this like is doing here. A debug leftover?

Reply:
Or maybe you wanted to check that this var is not visible? But it should be a different var name/value.

If you want a different name, what about runc_env or similar?

Reply:
Only StartContainer hook is running in the container.